third_party/HugeCTR/gpu_cache/include/nv_gpu_cache.hpp

```diff
@@ -28,8 +28,12 @@
 #endif
 
 #define SET_ASSOCIATIVITY 2
-#ifdef __HIPCC__
-#define SLAB_SIZE warpSize
+// TODO: Properly for portable HIP code, this should be determined at runtime,
+// but there's a lot of code that assumes this is a compile-time constant, so
+// for now we're hardcoding it. See
+// https://rocm.docs.amd.com/projects/HIP/en/latest/how-to/hip_cpp_language_extensions.html#warpsize
+#ifdef DGL_USE_HIP
+#define SLAB_SIZE 64
```

Comment:
FYI for future work, I actually have the fix for this, I just never landed it because there was weirdness about what was happening with `warpSize` and `__AMDGCN_WAVEFRONT_SIZE` upstream. The planned removal of those as compile-time constants has proceeded, so to adapt that PR, we'd probably need to create our own.

Reply:
Dropping a link to you fix for this:

https://github.com/nod-ai/dgl/pull/43